plugins/katello/nightly/installation/clients.md

```diff
@@ -58,6 +60,20 @@ yum install -y https://fedorapeople.org/groups/katello/releases/yum/{{ page.vers
 {% endhighlight %}
 </div>
 
+<div id="sles12" style="display:none;" markdown="1">
+{% highlight bash %}
+rpm -Uvh https://fedorapeople.org/groups/katello/releases/yum/{{ page.version }}/client/sles12/x86_64/katello-client-repos-latest.rpm
```

Comment:
This rpm doesn't exist (yet). Should it?

Reply:
It will as soon as the nightly nightly pipeline is passing, they are available at  http://koji.katello.org/releases/yum/katello-nightly/client/sles12/x86_64/ and http://koji.katello.org/releases/yum/katello-nightly/client/sles11/x86_64/   for now